sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenByAllPossible.java

```diff
@@ -0,0 +1,45 @@
+package com.saucelabs.appium.page_object.android;
+
+import io.appium.java_client.MobileElement;
+
+import java.util.List;
+
+import io.appium.java_client.android.AndroidElement;
+import io.appium.java_client.pagefactory.AndroidFindAll;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+
+/**
+ * 
+ * @author TikhomirovSergey
```

Comment:
In general we don't use author tags. It's common for many people to contribute using github and adding everyones name to an author tag doesn't work. Git is the best place to record authorship information.